Description: Enable subpixel rendering to provide LCD colour filtering.
Author: Steve Langasek <vorlon@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-29

--- a/devel/ftoption.h
+++ b/devel/ftoption.h
@@ -124,7 +124,7 @@
    * When this macro is not defined, FreeType offers alternative LCD
    * rendering technology that produces excellent output.
    */
-/* #define FT_CONFIG_OPTION_SUBPIXEL_RENDERING */
+#define FT_CONFIG_OPTION_SUBPIXEL_RENDERING
 
 
   /**************************************************************************
